graveyard.json

```diff
@@ -1,4 +1,12 @@
 [
+  {
+    "dateClose": "2020-05-12",
+    "dateOpen": "2019-06-10",
+    "description": "App used to find group activities with others who share your interests on Shoelace",
+    "link": "https://shoelace.nyc/",
```

Comment:
I think the link is supposed to be to some article which explains the death of the Google service. In this case, you can use https://docs.google.com/document/d/15NnvcvNtGdcpm3UMucwbF-V5TLjGur3sqkwfF8W6Fb0/edit which is Google's official notice.

Reply:
Let's actually link to an independent source. It's important as Google's links tend to 404 over time.

https://9to5google.com/2020/04/28/shoelace-area-120-shutdown/

Reply:
Now it's done!